NetworkingSampleApp/NetworkingSampleApp/API/SampleAuthorizationManager.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  SampleAuthorizationManager.swift
+//  NetworkingSampleApp
+//
+//  Created by Dominika Gajdová on 20.12.2022.
+//
+
+import Networking
+import Foundation
+
+final class SampleAuthorizationManager: AuthorizationManaging {
+    let storage: AuthorizationStorageManaging = SampleAuthorizationStorageManager()
+    
+    private let apiManager: APIManager = {
```

Comment:
I agree, I will do it once I add more sample scenarios to the app. However the `AuthorizationManager` needs it's own `APIManager` to handle the refresh token call, otherwise the `AuthorizationTokenInterceptor` would get stuck in a loop. I am actually thinking of adding an `APIManager` property to `AuthorizationManaging` with an explanation.

Reply:
yes, we exactly use it as expected, so you might have 2 managers or more in app with different setup. Reason - auth API may run different, you may need to connect to some 3rd party rest API etc